src/native.js

```diff
@@ -449,6 +449,9 @@ export function getAllAssetsMessage(data, adObject, {getNativeReq = getNativeReq
  * appropriate for sending in adserver targeting or placeholder replacement.
  */
 function getAssetValue(value) {
+  // as typeof null is 'object' this case is problematic in the next condition
+  if (value === null) value = '';
```

Comment:
I agree with @dgirardi that it's easier to reason about if the whole function body is just `return value?.url || value`.